tests/metrics/test_sensitivity.py

```diff
@@ -63,6 +69,34 @@ def test_basic_sensitivity_max_multiple(self):
             sa.attribute, (input1, input2), [0.0] * 20, max_examples_per_batch=60
         )
 
+    def test_basic_sensitivity_max_multiple_gradshap(self):
+        model = BasicModel2()
+        gs = GradientShap(model)
+
+        input1 = torch.tensor([3.0] * 5)
+        input2 = torch.tensor([1.0] * 5)
+
+        baseline1 = torch.arange(0, 2).float()
+        baseline2 = torch.arange(0, 2).float()
+
+        self.setUp()
+        self.sensitivity_max_assert(
+            gs.attribute,
+            (input1, input2),
+            [0.6082, 0.5440, 0.3457, 2.6726, 0.3360],
```

Comment:
Good point! I made inputs very close to baselines so that randomly chosen path between those two will land a point that will lead to zero sensitivity.